include/mlir/Dialect/SPIRV/SPIRVOps.td

```diff
@@ -153,6 +153,52 @@ def SPV_CompositeExtractOp : SPV_Op<"CompositeExtract", [NoSideEffect]> {
 
 // -----
 
+def SPV_CompositeInsertOp : SPV_Op<"CompositeInsert", [NoSideEffect]> {
```

Comment:
Could you create a new file, `SPIRVCompositeOps.td`, and put the definition there to follow spec organization?